src/main/java/be/appwise/core/networking/proxyman/Configuration.kt

```diff
@@ -0,0 +1,20 @@
+package be.appwise.core.networking.proxyman
+
+class Configuration(
```

Comment:
I followed the placement of files like the official Proxyman Atlantis library. We maybe want to let them publish it. So i kept the same placement

Reply:
Hmm, I think it's not bad following their designs and file structure to some extent.
But in some cases it can be helpful to improve on others their designs.